scripts/sortTermsAlphabetically.gradle

```diff
@@ -0,0 +1,108 @@
+import static groovy.io.FileType.FILES
+import java.util.regex.Matcher
+
+class Term {
+    String term
+    String captionEN
+    String captionDE
+    String includeFragment
+
+    Term(String term, String captionEN, String captionDE, includeFragment) {
+        this.term = term
+        this.captionEN = captionEN
+        this.captionDE = captionDE
+        this.includeFragment = includeFragment
+    }
+}
+
+task sortTermsAlphabetically(
+        description: 'collect terms and create an alphabetically orders structure',
+        group: 'isaqb-glossary'
+) {
+    doLast {
+        def terms = createTermsContent()
+
+        sortTermsEN(terms)
+        writeTermsStructure(terms, "EN")
+
+        sortTermsDE(terms)
+        writeTermsStructure(terms, "DE")
+    }
+
+    ext.createTermsContent = {
+        def terms = collectTerms()
+        List alphabet = ('A'..'Z').collect { it }
+        alphabet.each(
+                letter -> terms.add(new Term(/==== $letter/, /==== $letter/, /==== $letter/, /=== $letter/))
+        )
+        return terms
+    }
+
+    ext.collectTerms = {
+        def termsFolder = new File(projectDir, '/docs/1-terms')
+        def terms = []
+
+        termsFolder.traverse(type: FILES) { file ->
+
+            if (file.name ==~ 'term.*[.](ad|adoc|asciidoc)$') {
```

Comment:
I guess we could simplify this even more to only accept adoc.

Reply:
Sure we could, but all this extensions are valid for ascii-doc files. 
I think it's not harmful to have this flexibility that one may never need. 
But the matching regex term is just nice :grin:

But if you insist, we can of course simplify the filter. Do you?

Reply:
It's alright. It's not wrong, easy to understand.

Reply:
:)